wetterdienst/provider/dwd/radar/api.py

```diff
@@ -55,6 +55,7 @@ def __init__(
         end_date: Optional[Union[str, datetime, timedelta]] = None,
         resolution: Optional[Union[str, Resolution, DwdRadarResolution]] = None,
         period: Optional[Union[str, Period, DwdRadarPeriod]] = None,
+        **kwargs,
```

Comment:
My personal opinion: Using **kwargs is real bad. Because there is not information what you can pass through . I think using **kwargs is for lazy programmers knowing what they do. 

Here you just expecting a "read_bufr" variable. I recommend to introduce a variable to make the usage clearly.

Reply:
I totally agree with you. My idea was that `read_bufr` would only be required if you want to actually get bufr data because we have other kinds of data available as well. However we may just use the arg `read_bufr` for now and later move this option to a wetterdienst config or something related e.g. 

`config.radar.read_bufr = True`

which we have somewhat planned for all kinds of meta arguments like `tidy`, `si_units`, `humanize`etc.